include/envoy/http/access_log.h

```diff
@@ -114,6 +114,16 @@ class RequestInfo {
    * Set whether the request is a health check request or not.
    */
   virtual void healthCheck(bool is_hc) PURE;
+
+  /**
+   * Set the downstream address
+   */
+  virtual void setDownstreamAddress(std::string address) PURE;
```

Comment:
But there is no reason to provide a setter on the interface. When will it ever be set at an interface level?